README.md

```diff
@@ -13,25 +13,19 @@ Article templates used within the Guardian’s next-generation iOS, Android and
 
 ## Developing
 * checkout the project in a separate directory, outside the iOS and the Android app.
-* copy config.sample.js to config.js and fill in the details
-    * `base.android` is the 'ArticleTemplate' path within the Android app, eg: `/Users/sandropaganotti/Projects/android-news-app/android-news-app/src/main/assets/templates/`
-    * `base.html` is the path where this repository has been checked out, eg: `/Users/sandropaganotti/Projects/mobile-apps-article-templates/`
 * run `yarn` to install dependencies.
-* run `yarn setup` to locally ignore build files so they are not checked into master
```

Comment:
You're right, we don't need this anymore, but the build files should be in .gitignore because we don't commit them?